src/app/hooks/useViewTracker/index.tsx

```diff
@@ -46,72 +47,25 @@ const getComponentViewTracker = (eventTrackingData?: EventTrackingData) => {
 
   const observer = useRef(null);
   const timer = useRef(null);
-  const [isInView, setIsInView] = useState(false);
+  const [componentHasComeIntoView, setcomponentHasComeIntoView] =
```

Comment:
Minor but `setcomponentHasComeIntoView` could be `setComponentHasComeIntoView`. Just a capital 'C' on the `component` word.